pkgs/by-name/ut/ut1999/package.nix

```diff
@@ -41,41 +63,50 @@ in stdenv.mkDerivation rec {
     stdenv.cc.cc
   ];
 
-  nativeBuildInputs = [
+  nativeBuildInputs = lib.optionals (!stdenv.isDarwin) [
```

Comment:
Nit:

```suggestion
  nativeBuildInputs = lib.optionals (stdenv.isLinux) [
```

Reply:
BTW also the parenthesis aren't required.